mailo/src/main/scala/mailo/http/MailgunClient.scala

```diff
@@ -159,7 +157,7 @@ class MailgunClient(
     name: String,
     `type`: ContentType,
     content: String,
-    transferEncoding: Option[String] = None
```

Comment:
hm? default removed?

Reply:
yes, it was unused (the compiler told me so)